base_delivery_carrier_label/readme/USAGE.rst

```diff
@@ -1,12 +1,26 @@
 ** How does it works ? **
 
 
-In picking UI a button "Shipping label" trigger label generation
-calling `action_generate_carrier_label()` in models/stock.picking.py
+In picking UI a button "Send to shipper" trigger label generation
+calling `send_to_shipper()` in models/stock.picking.py
 
 
 ** How to implement my own carrier ? **
 
 
-Override `generate_shipping_labels()` which is called by previous method
-in the same file.
+Define a method `carrier_delivery_type_send_shipping()` which is called by _send_shipping native method.
```

Comment:
`carrier_delivery_type_send_shipping()`

If I get it right here `carrier_delivery_type` is to be replaced by some other string?
(By the way it's delivery.carrier and not carrier.delivery)

Would it make it clearer if we wrote something like: `{carrier}_send_shipping()`

```
Define a method `{carrier}_send_shipping()` which is called by _send_shipping native method.
Where `{carrier}` is a `delivery.carrier` type.
```

(I think `carrier` is enough in this context)

Reply:
Fixed
I guess it is clearer indeed, thanks